docs/env-k3s.md

```diff
@@ -67,11 +67,11 @@ This is something we intend figuring out in the near future.
 
 ### From the internet
 
-One can easily setup k3s with s3gw from the internet, by running
+One can easily set up k3s with s3gw from the internet, by running
 
 ```shell
 curl -sfL \
-  https://raw.githubusercontent.com/aquarist-labs/s3gw-tools/main/k3s/setup.sh \
+  https://raw.githubusercontent.com/aquarist-labs/s3gw-tools/env/setup.sh \
```

Comment:
this URL does not exist. I believe what you want is `https://raw.githubusercontent.com/aquarist-labs/s3gw-tools/main/env/setup.sh`.